README.md

```diff
@@ -1,2 +1,27 @@
-# support
-For questions on how to get IPFS up and running smoothly
+# Support
+
+For questions on how to get IPFS up and running smoothly.
```

Comment:
indeed, though perhaps the flow should be to post in support first, as it would build an easier-to-search bank of common problems/answers.